libdevice/.clangd

```diff
@@ -0,0 +1,5 @@
+CompileFlags:
```

Comment:
Can I add a clangd configuration file which is very helpful for clangd's user?
This file will help to enable autocomplete when editing libdevice project.

Reply:
people may have different preference over clangd setting, can we just put this in local clangd config file?

Reply:
Okay, I add this file to .gitignore.

Reply:
Removed